ubuntu_desktop_installer/lib/l10n/intl_oc.arb

```diff
@@ -0,0 +1,87 @@
+{
+  "@@last_modified": "2020-11-20T17:25:14.036182",
+  "Ubuntu Desktop Installer": "Programa d’installacion del burèu Ubuntu",
+  "@Ubuntu Desktop Installer": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Go Back": "Tornar",
+  "@Go Back": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Continue": "Contunhar",
+  "@Continue": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Restart": "Reaviar",
+  "@Restart": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "Try or install": "Tornar ensajar o installar",
```

Comment:
I don't speak Occitan (but I do speak Catalan and French, so I can understand most of it). This "Tornar" looks out of place, is this correct?

Reply:
yes right, I had in mind "retry" somehow :)